juspay/hyperswitch-control-center
# chore: Enhance transaction views

## Type of Change




## Description
Introduced None view in Views , so that no view will be choosen on UI in case of custom status filters applied by user 

Filters and Views are  made to be in sync now, if you select a  status filter for which view exists , then that view will be shown as selected in UI

This change work similar for Orders, Refunds and Disputes


![image](https://github.com/user-attachments/assets/5c25b7a0-b1c8-4819-9341-e965db6e756d)
![image](https://github.com/user-attachments/assets/d7fbe6ba-a9cc-4938-9e1d-bf754f00a067)
![image](https://github.com/user-attachments/assets/1d7db8b7-0c00-4176-aed9-10c16ba743d7)
![image](https://github.com/user-attachments/assets/381e3b06-c750-4b13-b669-2cc9218f417d)


## Motivation and Context



## How did you test it?

refer description

1. go to payments
2. apply any view from top
3. change status filters
4. if multiple status filters choosen, no view is shown as selected
5. in case of single status filter choosen, and if that  view exists in top, we will show that view as choosen


## Where to test it?


## Checklist
